config/prow/cluster/monitoring/mixins/grafana_dashboards/plank.jsonnet

```
@@ -23,7 +23,7 @@ dashboard.new(
   template.new(
     'cluster',
     'prometheus',
-    'label_values(prowjobs{job="plank"}, cluster)',
+    'label_values(prowjobs{job="prow-controller-manager"}, cluster)',
```

So we don't lose history
```suggestion
    'label_values(prowjobs{job=~"plank|prow-controller-manager"}, cluster)',
```

---

(this is in multiple places I just felt like it'd be easier for you to search-replace than me to make multiple suggetions)